diff --git a/swift/utils/build-presets.ini b/swift/utils/build-presets.ini
index 1d1a17b1f0b..229ff2a19d6 100644
--- a/swift/utils/build-presets.ini
+++ b/swift/utils/build-presets.ini
@@ -916,6 +916,11 @@ mixin-preset=
     buildbot_linux,no_assertions
     mixin_buildbot_linux,no_test
 
+[preset: buildbot_amazon_linux_2023,no_assertions,no_test]
+skip-early-swift-driver
+mixin-preset=
+    buildbot_linux,no_assertions
+    mixin_buildbot_linux,no_test
 
 [preset: buildbot_linux_crosscompile_android,tools=RA,stdlib=RD,build]
 mixin-preset=