diff -Naur swift-orig/cmake/modules/AddSwift.cmake swift/cmake/modules/AddSwift.cmake
--- swift-orig/cmake/modules/AddSwift.cmake	2021-01-11 09:24:03.000000000 -0600
+++ swift/cmake/modules/AddSwift.cmake	2021-01-14 16:07:26.118061686 -0600
@@ -288,7 +288,7 @@
     if(SWIFT_HOST_VARIANT_ARCH STREQUAL x86_64)
       # this is the minimum architecture that supports 16 byte CAS, which is
       # necessary to avoid a dependency to libatomic
-      target_compile_options(${target} PRIVATE -march=core2)
+      target_compile_options(${target} PRIVATE -mcx16)
     endif()
   endif()
 
diff -Naur swift-orig/stdlib/cmake/modules/AddSwiftStdlib.cmake swift/stdlib/cmake/modules/AddSwiftStdlib.cmake
--- swift-orig/stdlib/cmake/modules/AddSwiftStdlib.cmake	2021-01-11 09:24:03.000000000 -0600
+++ swift/stdlib/cmake/modules/AddSwiftStdlib.cmake	2021-01-14 16:06:49.905752127 -0600
@@ -306,7 +306,7 @@
   if("${CFLAGS_SDK}" STREQUAL "LINUX")
     if(${CFLAGS_ARCH} STREQUAL x86_64)
       # this is the minimum architecture that supports 16 byte CAS, which is necessary to avoid a dependency to libatomic
-      list(APPEND result "-march=core2")
+      list(APPEND result "-mcx16")
     endif()
   endif()
 
diff -Naur swift-orig/utils/build-presets.ini swift/utils/build-presets.ini
--- swift-orig/utils/build-presets.ini	2021-01-11 09:24:03.000000000 -0600
+++ swift/utils/build-presets.ini	2021-01-14 15:42:31.063141040 -0600
@@ -771,7 +771,6 @@
 libicu
 libcxx
 
-build-ninja
 install-llvm
 install-swift
 install-lldb
@@ -787,10 +786,6 @@
 build-swift-static-sdk-overlay
 build-swift-stdlib-unittest-extra
 
-# Executes the lit tests for the installable package that is created
-# Assumes the swift-integration-tests repo is checked out
-
-test-installable-package
 
 # Build the benchmarks against the toolchain.
 toolchain-benchmarks
